db/migrate/20210426191308_replace10_x_in_alexandria_convention.rb

```diff
@@ -0,0 +1,22 @@
+class Replace10XInAlexandriaConvention < Mongoid::Migration
+  # from https://cloud.google.com/bigquery/docs/updating-data#updating_data
+  def self.up
+    client = BigQueryClient.new.client
+    [CellMetadatum::BIGQUERY_DATASET, 'cell_metadata_test'].each do |dataset_name|
+      dataset = client.dataset(dataset_name)
+      if dataset.present?
+        col_name = 'library_preparation_protocol__ontology_label'
+        # query will match all instances of library_preparation_protocol__ontology_label beginning with the
+        # substring '10X ' and replace it with '10x ', e.g. "10X 3' v2 sequencing" => "10x 3' v2 sequencing"
+        update_query = "UPDATE #{CellMetadatum::BIGQUERY_TABLE}"
+        replace_clause = "SET #{col_name} = REGEXP_REPLACE(#{col_name}, r\"10X \", \"10x \")"
```

Comment:
That's actually a great point - that was not intentional.  It won't have any effect because all of the possible values for 10x sequencing entries in EFO start with `10x`, but you are totally correct in that the statement above doesn't exactly do what we want.

I found another issue as well in that while we update the data in BQ, it doesn't update the search facet filters because we didn't call `SearchFacet.update_all_facet_filters`.  I'm going to reopen this and fix both issues.